### PR TITLE
Fix typos, broken link & better prometheus conf

### PR DESCRIPTION
**Pull request by coulof**
# Description
This PR fixes minor syntax issue, correct some filename references and give an improved Prometheus config (doesn't cancel kubernetes scrape metrics anymore), update some versions and fix the VGsnapshot yaml (tested).

It also fix a broken link referenced in the resiliency module and align better the tables

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Comment by tdawe** [comment]
Can you create a branch off of `release-1.2` and then target the PR at that branch. Thanks

**Review by shanmydell — CHANGES_REQUESTED** [review]
Renaming a link needs, link reference to be updated for FAQ page

**Comment by coulof** [comment]
@tdawe, I solved the conflicts with the `release-1.2` branch

@shanmydell, sorry I don't understand the comment. The link in the FAQ was broken due to the renaming of the title.

**Comment by shanmydell** [comment]
@coulof : [How do I uninstall or disable a Container Storage Module?](#how-do-i-uninstall-or-a-disable-a-module) As you have renamed the text in the question , the link on the side also will be updated, we have to change the reference of link as well

File: content/docs/FAQ/_index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ weight: 2
 - [Where do I start with Dell Container Storage Modules (CSM)?](#where-do-i-start-with-dell-container-storage-modules-csm)
 - [Is the Container Storage Module XYZ available for my array?](#is-the-container-storage-module-xyz-available-for-my-array)
 - [What are the prerequisites for deploying Container Storage Modules?](#what-are-the-prerequisites-for-deploying-container-storage-modules)
-- [How do I uninstall or disable a Container Storage Module?](#how-do-i-uninstall-or-a-disable-a-module)
+- [How do I uninstall or a disable a module?](#how-do-i-uninstall-or-a-disable-a-module)
```
Comment (AronAtDell) [inline content/docs/FAQ/_index.md:12]: remove "a" before "disable"

File: content/docs/authorization/_index.md [hunk 2]
```diff
@@ -96,4 +99,4 @@ Tenants of CSM for Authorization can use the token provided by the Storage Admin
 4) Tenant Admin inputs the Token into their Kubernetes cluster as a Secret.
 5) Tenant Admin updates CSI driver with CSM Authorization sidecar module.
 
-![CSM for Authorization Workflow](./design2.png "CSM for Authorization Workflow")
\ No newline at end of file
+![CSM for Authorization Workflow](./design2.png "CSM for Authorization Workflow")
```
Comment (AronAtDell) [inline content/docs/authorization/_index.md:102]: What is the change here?

Comment (coulof) [reply content/docs/authorization/_index.md:102]: There was a trailing EOL

**Comment by coulof** [comment]
> @coulof : [How do I uninstall or disable a Container Storage Module?](#how-do-i-uninstall-or-a-disable-a-module) As you have renamed the text in the question , the link on the side also will be updated, we have to change the reference of link as well

Actually, the title has been renamed before, I just updated the link. I can change it back to what it used to be if you prefer or maybe you can point to where should happen this other change.

**Comment by coulof** [comment]
@tdawe  & @shanmydell, do you mind reviewing once again ?

Thanks !

**Review by shanmydell — DISMISSED** [review]
LGTM

**Comment by shanmydell** [comment]
> @tdawe & @shanmydell, do you mind reviewing once again ?
> 
> Thanks !

Done.

**Review by maxan98 — DISMISSED** [review]

**Review by shanmydell — APPROVED** [review]

**Review by tdawe — APPROVED** [review]

**Pull request merged**